lang/main-ca.json

```diff
@@ -464,6 +464,32 @@
         "newDeviceAudioTitle": "S'ha detectat un aparell d'àudio nou",
         "newDeviceAction": "Usa"
     },
+"participantsPane": {
```

Comment:
Can you fix the indentation here please?

Reply:
Sure